StripeIdentity/StripeIdentity/Source/NativeComponents/Views/IdentityFlowView.swift

```diff
@@ -114,6 +118,12 @@ class IdentityFlowView: UIView {
         return buttonBackgroundView
     }()
 
+    private let buttonTopBackgroundView: UIView = {
+        let buttonTopBackgroundView = UIView()
+        buttonTopBackgroundView.backgroundColor = .systemBackground
```

Comment:
`systemBackground` _might_ be different in dark/light mode? so just adding a note in case that's important to check

Reply:
yes, this is expected and also verified in darkmode, we also use the same property for `buttonBackgroundView` at line 117